### PR TITLE
Patch Release v0.56.1

### PR DESCRIPTION
**Pull request by chitrangpatel**
This updates the latest release to the patch release.

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] [pre-commit](https://github.com/tektoncd/pipeline/blob/main/DEVELOPMENT.md#install-tools) Passed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [ ] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```
/kind misc

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
/lgtm

File: releases.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Further documentation available:
 ## Release
 ### v0.56 (LTS)
 
-- **Latest Release**: [v0.56.0][v0.56-0] (2024-01-19) ([docs][v0.56-0-docs], [examples][v0.56-0-examples])
+- **Latest Release**: [v0.56.1][v0.56-1] (2024-02-13) ([docs][v0.56-1-docs], [examples][v0.56-1-examples])
```
Comment (JeromeJu) [inline releases.md:45]: NIT: we might also need to update https://github.com/tektoncd/pipeline/pull/7663/files#diff-1d2b91d957f8612017f196e6cc2e83c33bcfe3fd348dc3619e9e9672e797d46fR190

i.e. following https://github.com/tektoncd/pipeline/commit/f990d8cb79762fe126681e71622ead8efb1f4ba8

Comment (chitrangpatel) [reply releases.md:45]: Agreed! Thanks for that! I submitted a new PR to fix this.

**Pull request merged**